src/main/java/everyos.browser.webicitybrowser/everyos/browser/webicitybrowser/util/TimeSystem.java

```diff
@@ -0,0 +1,26 @@
+package everyos.browser.webicitybrowser.util;
+
+// TODO: Better name, and not TimeUtil, because everyos.browser.javadom.imp.TimeUtil
+public class TimeSystem {
+
+    private static long lastTime = -1;
+    private static long delta = -1;
+
+    public static void reset() {
```

Comment:
Static is a no-go here. What happens if we open multiple windows?

Reply:
Let it stay like that, I'll think of a solution later

Reply:
Ok. Mark it as TODO at least though.